src/middlewared/middlewared/etc_files/local/openvpn/server/perms.py

```diff
@@ -0,0 +1,29 @@
+import errno
+import os
+
+from pathlib import Path
+
+from middlewared.service import CallError
+
+
+LOGS_FOLDER = '/var/log/openvpn'
+
+
+def fix_perms(middleware):
+    if nobody_user := middleware.call_sync('user.query', [('username', '=', 'nobody')]):
+        nobody_uid = nobody_user[0]['uid']
+        nobody_gid = nobody_user[0]['group']['bsdgrp_gid']
+    else:
+        raise CallError('Unable to locate "nobody" user', errno=errno.ENOENT)
+
+    log_dir = Path(LOGS_FOLDER)
+    log_dir.mkdir(parents=True, exist_ok=True)
+    for path_attr in (
```

Comment:
Why are you still doing this complicated logic? A simple for loop will suffice....

Reply:
It is a simple for loop ?
Ownership needs to be changed for the directory as well so it needs to be in the for loop as well

Reply:
I don't mind either ways but curious why write the same chown statement 2 times instead of having log dir in the for loop as well ?